opm/parser/eclipse/EclipseState/InitConfig/Equil.hpp

```diff
@@ -0,0 +1,61 @@
+#ifndef OPM_EQUIL_HPP
+#define OPM_EQUIL_HPP
+
+namespace Opm {
+
+    class DeckKeyword;
+    class DeckRecord;
+
+    class EquilRecord {
+        public:
+            EquilRecord( const DeckRecord& );
+
+            double datumDepth() const;
+            double datumDepthPressure() const;
+            double waterOilContactDepth() const;
+            double waterOilContactCapillaryPressure() const;
+            double gasOilContactDepth() const;
+            double gasOilContactCapillaryPressure() const;
+
+            int liveOilInitProcedure() const;
+            int wetGasInitProcedure() const;
+            int initializationTargetAccuracy() const;
+            int compositionalInitializationProcedure() const;
+
+            bool useSaturationPressure() const;
+
+        private:
+            double datum_depth;
+            double datum_depth_ps;
+            double water_oil_contact_depth;
+            double water_oil_contact_capillary_pressure;
+            double gas_oil_contact_depth;
+            double gas_oil_contact_capillary_pressure;
+
+            int live_oil_init_proc;
+            int wet_gas_init_proc;
+            int init_target_accuracy;
+    };
+
+    class Equil {
+        public:
+            using const_iterator = std::vector< EquilRecord >::const_iterator;
+
+            Equil() = default;
+            Equil( const DeckKeyword& );
```

Comment:
Do you mean the constructor? I have no strong opinion on it for cases like this, but it can be made explicit for sure.